LanguageApp/app/screens/ProviderScenarioScreen.js

```diff
@@ -40,22 +46,59 @@ function ProviderScenarioScreen({ route }) {
     setAnswerAudio(data);
   };
 
-  // Submission data to be passed to Firebase
-  const translatedScenario = {
-    promptTranslation: cpPrompt,
-    answerTranslation: cpAnswer,
-    promptRecording: promptAudioLink,
-    answerRecording: answerAudioLink,
-    translatorID: 11111111,
-  };
+    // Submission data to be passed to Firebase
+    const translatedScenario = {
+      promptTranslation: cpPrompt,
+      answerTranslation: cpAnswer,
+      promptRecording: promptAudioLink,
+      answerRecording: answerAudioLink,
+      translatorID: 11111111,
+    };
```

Comment:
It seems that this is no longer needed with your implementation. Can delete if no longer necessary.